enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/entity/OnboardingOtpEntity.java

```diff
@@ -61,6 +61,13 @@ public class OnboardingOtpEntity implements Serializable {
     @JoinColumn(name = "process_id", referencedColumnName = "id", nullable = false)
     private OnboardingProcessEntity process;
 
+    /**
+     * Not-null only for {@link OtpType#USER_VERIFICATION} when limit is needed to count not over the whole process.
+     */
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Good idea. We should probably review whether the `LAZY` fetch type could be added elsewhere, too.